content/docs/apps/govcloud.md

```diff
@@ -83,18 +81,15 @@ Any labor involved from your project team is the responsibility of your project.
 
 #### Process
 
-1. Fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1).
-    * Specify that the org should be created in `GovCloud`.
-    * An admin will confirm the information, and [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you.
+1. [Get access to the GovCloud environment](https://account.fr.cloud.gov/signup).
+1. Request the creation of your org in the GovCloud environment:
+    * If you are in 18F/TTS at GSA, fill out the [organization request form](https://docs.google.com/a/gsa.gov/forms/d/e/1FAIpQLSd4HmcGfJW3EBnpewTFDD-urRFPp1LN0DcwNB_FxZgUn8ho9g/viewform?c=0&w=1). Specify that the org should be created in `GovCloud`. An admin will confirm the information, [create the org]({{< relref "docs/ops/create-org.md#non-sandboxes" >}}) for you, and notify you. This should happen within one business day.
```

Comment:
Would like to make sure they tag their subject line with something like `[GovCloud Org Creation]`